### PR TITLE
Disabled realtime spellcheck error

### LINKED ISSUE #7018: Problems with spell check Macintosh
Trying to use Spell Check under MacOs Catalina Rstudio Version 1.3.959

The following error pops up:

check spelling An error has occurred: (TypeError) : Cannot read property 'check' of undefined

The erros happens whichever language I choose for checking

### LINKED ISSUE #7057: Check Spelling with real-time spellcheck off throws error
### System details

    RStudio Edition : Desktop or Server
    RStudio Version : 1.3.1008
    OS Version      : Centos7 (Server), macOS (Desktop)
    R Version       : 3.6.2 server, 3.6.1 desktop

### Steps to reproduce the problem
- turn off real-time spellcheck in Global Options / Spelling
- quit and restart IDE
- open an existing document that supports spell-check such as an RMD file
- F7 or toolbar spellcheck icon or Edit / Check Spelling

### Describe the problem in detail
- get error dialog with "Cannot read property "check" of undefined"

screenshot of error dialog

### Describe the behavior you expected
- no such error

- [ ] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [ ] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [ ] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [ ] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

### PR DESCRIPTION
**Pull request by adamconroy**
We use Typo.js for even non-realtime spellchecking now but we were trying to over-optimize loading the dictionaries and an edge case was missed.

Fixes #7057
Fixes #7018

### CODE REVIEW

**Comment by gtritchie** [comment]
We don’t use the 1.3-patch branch any more, should target v1.3

**Comment by gtritchie** [comment]
Oops, I didn't mean to close this!

File: NEWS-1.3-patch.md [hunk 1]
```diff
@@ -27,6 +27,8 @@
 - Fix auto-activation of JAWS screen reader virtual cursor in Console Output region (#6884)
 - Announce text of warnings bar via screen reader (#6963)
 - Fix issue where R_LIBS_SITE could be forcibly set empty, overriding the value in /etc/R/REnviron (#6982)
+- Fix sign out from the Admin Dashboard when behind a path-rewriting proxy (Pro #1709)
+- Fix "Login as user" from the Admin Dashboard when using Launcher sessions (Pro #1710)
```
Comment (jmcphers) [inline NEWS-1.3-patch.md:31]: These changes look unrelated (and this file doesn't even exist on the 1.3 branch) so you will probably need to reapply.

**Comment by adamconroy** [comment]
I actually am going to close this and just reopen with the latest 1.3 branch as the base. I changed it and noticed that conflict. I didn't know that 1.3-patch was defunct.
